### PR TITLE
add captions for 2022-06-09

### LINKED ISSUE #4: [transcript] 2022-06-09
Assigned to: @fsb4000

### PR DESCRIPTION
**Pull request by fsb4000**
Fixes #4

I will watch again tomorrow and collect all places where I am unsure about correctness.

### CODE REVIEW

**Comment by fsb4000** [comment]
Could you watch these links for ~10 seconds each and check correctness?

00:11:48.506 --> 00:11:51.480
https://youtu.be/i2wzjRJjuKY?t=706

00:17:39.478 --> 00:17:42.334
https://youtu.be/i2wzjRJjuKY?t=1059

00:23:11.816 --> 00:23:13.676
https://youtu.be/i2wzjRJjuKY?t=1389

00:29:10.974 --> 00:29:13.628
https://youtu.be/i2wzjRJjuKY?t=1751

00:29:36.110 --> 00:29:39.734
https://youtu.be/i2wzjRJjuKY?t=1778

00:30:11.704 --> 00:30:13.768
https://youtu.be/i2wzjRJjuKY?t=1812

00:37:52.820 --> 00:37:57.529
https://youtu.be/i2wzjRJjuKY?t=2273

00:38:32.120 --> 00:38:33.410
https://youtu.be/i2wzjRJjuKY?t=2312

00:42:26.330 --> 00:42:28.605
https://youtu.be/i2wzjRJjuKY?t=2546

00:45:00.090 --> 00:45:00.388
https://youtu.be/i2wzjRJjuKY?t=2700

00:52:06.500 --> 00:52:09.140
https://youtu.be/i2wzjRJjuKY?t=3126

File: transcripts/2022/2022-06-09/edited.vtt [hunk 1]
```diff
@@ -0,0 +1,5254 @@
...
+is we could just say this is a
+
+00:29:54.788 --> 00:29:56.096
+corner case for like iota views.
+
+00:29:56.100 --> 00:29:58.020
+Merge this as is and then
+
+00:29:58.020 --> 00:29:59.608
+catch it up in 1206.
```
Comment (strega-nil-ms) [inline transcripts/2022/2022-06-09/edited.vtt:2232]:
```suggestion
patch it up in 1206.
```

File: transcripts/2022/2022-06-09/edited.vtt [hunk 2]
```diff
@@ -0,0 +1,5254 @@
...
+which hopefully I mean.
+
+00:30:11.704 --> 00:30:13.768
+On the other hand it is vector
+
+00:30:13.768 --> 00:30:15.188
+if you don't want to.
+
+00:30:15.190 --> 00:30:17.070
+Find their own range types
```
Comment (strega-nil-ms) [inline transcripts/2022/2022-06-09/edited.vtt:2259]:
```suggestion
They can define their own range types
```

File: transcripts/2022/2022-06-09/edited.vtt [hunk 3]
```diff
@@ -0,0 +1,5254 @@
...
+with extremely large iota views,
+
+00:30:10.560 --> 00:30:11.704
+which hopefully I mean.
+
+00:30:11.704 --> 00:30:13.768
+On the other hand it is vector
+
+00:30:13.768 --> 00:30:15.188
+if you don't want to.
```
Comment (strega-nil-ms) [inline transcripts/2022/2022-06-09/edited.vtt:2256]:
```suggestion
if you don't want to.
&gt;&gt; Casey: Or user-defined range types.
```

File: transcripts/2022/2022-06-09/edited.vtt [hunk 4]
```diff
@@ -0,0 +1,5254 @@
...
+It's on my other screen, which is
+
+00:29:26.032 --> 00:29:27.760
+why you can't see it on the the view.
+
+00:29:27.760 --> 00:29:32.370
+OK, so you've generalized in your code.
+
+00:29:32.370 --> 00:29:33.980
+&gt;&gt; Stephan: Convert size to handle this.
```
Comment (strega-nil-ms) [inline transcripts/2022/2022-06-09/edited.vtt:2198]:
```suggestion
Convert size to handle this.
```

File: transcripts/2022/2022-06-09/edited.vtt [hunk 5]
```diff
@@ -0,0 +1,5254 @@
...
+because it's just some random
+
+00:37:49.490 --> 00:37:51.200
+branch rather than our own.
+
+00:37:51.200 --> 00:37:52.820
+OK, but we got the citation.
+
+00:37:52.820 --> 00:37:57.529
+&gt;&gt; Casey: It is unfortunate
```
Comment (strega-nil-ms) [inline transcripts/2022/2022-06-09/edited.vtt:2697]:
```suggestion
&gt;&gt; Casey: That's unfortunate... I hit y to make sure it was a permalink
```

File: transcripts/2022/2022-06-09/edited.vtt [hunk 6]
```diff
@@ -0,0 +1,5254 @@
...
+&gt;&gt; Stephan: yeah, the fork is not wide itself.
```
Comment (strega-nil-ms) [inline transcripts/2022/2022-06-09/edited.vtt:2698]:
```suggestion
&gt;&gt; Stephan: yeah, the fork is not... the repo itself.
```

File: transcripts/2022/2022-06-09/edited.vtt [hunk 7]
```diff
@@ -0,0 +1,5254 @@
...
+
+00:37:57.529 --> 00:37:59.285
+&gt;&gt; Stephan: Yeah, Casey recommends just
```
Comment (strega-nil-ms) [inline transcripts/2022/2022-06-09/edited.vtt:2701]:
```suggestion
Yeah, Casey recommends just
```

File: transcripts/2022/2022-06-09/edited.vtt [hunk 8]
```diff
@@ -0,0 +1,5254 @@
...
+So this ensures if you met the classic
+
+00:42:11.310 --> 00:42:13.078
+requirement we will still activate it.
+
+00:42:13.080 --> 00:42:14.538
+This is safe.
+
+00:42:14.538 --> 00:42:18.180
+I like this if we don't have concepts
```
Comment (strega-nil-ms) [inline transcripts/2022/2022-06-09/edited.vtt:3034]:
```suggestion
I like this. If we don't have concepts
```

File: transcripts/2022/2022-06-09/edited.vtt [hunk 9]
```diff
@@ -0,0 +1,5254 @@
...
+Like insert forward range.
+
+00:44:52.190 --> 00:44:54.038
+It's a large function,
+
+00:44:54.038 --> 00:44:56.810
+it's doing a lot of work.
+
+00:44:56.810 --> 00:44:57.750
+&gt;&gt; Nicole: That's very good point.
```
Comment (strega-nil-ms) [inline transcripts/2022/2022-06-09/edited.vtt:3238]:
```suggestion
&gt;&gt; Nicole: That's a very good point.
```

File: transcripts/2022/2022-06-09/edited.vtt [hunk 10]
```diff
@@ -0,0 +1,5254 @@
...
+
+00:44:57.750 --> 00:45:00.088
+&gt;&gt; Nicole: I did not notice how big that function was.
+
+00:45:00.090 --> 00:45:00.388
+&gt;&gt; Stephan: Yeah,
+
+00:45:00.388 --> 00:45:03.070
+&gt;&gt; Nicole: they're my context is it takes having a lot?
```
Comment (strega-nil-ms) [inline transcripts/2022/2022-06-09/edited.vtt:3247]:
```suggestion
&gt;&gt; Nicole: Never mind I take it back
&gt;&gt; Stephan: the context is hiding a lot
```

File: transcripts/2022/2022-06-09/edited.vtt [hunk 11]
```diff
@@ -0,0 +1,5254 @@
...
+
+00:51:58.520 --> 00:52:02.870
+&gt;&gt; Stephan: We can check 
+&gt;&gt; Casey: so it would be consistently
+
+00:52:02.870 --> 00:52:03.420
+&gt;&gt; Casey: wrong.
+
+00:52:06.500 --> 00:52:09.140
+&gt;&gt; Casey: Or do we always define another one in
```
Comment (strega-nil-ms) [inline transcripts/2022/2022-06-09/edited.vtt:3808]:
```suggestion
&gt;&gt; Casey: Or do we always define... I know there's one in
```

**Review by strega-nil-ms — CHANGES_REQUESTED** [review]
Got some minor comments, and I've marked what I heard in the video :)

Thanks so, so much
oh my god thank you so muchhhh

**Comment by strega-nil-ms** [comment]
Thank you!

**Pull request merged**